org.gnome.HexGL.json

```diff
@@ -85,8 +74,11 @@
             "sources" : [
                 {
                     "type" : "git",
-                    "url" : "https://github.com/alexlarsson/gnome-hexgl",
-                    "tag" : "0.2.0"
+                    "url" : "https://github.com/alexlarsson/gnome-hexgl.git"
```

Comment:
you need an explicit revision.

Reply:
0.2.0 is the latest but that doesn't build I pasted the logs in the first post. Any idea?

Reply:
Oh, I misread revision as version...

Reply:
did you file a bug about not building? Also you updated a depenency with what I think, according to the version number, might be an incompatible revision.

Reply:
The error looks weird and unreleated to the dependencies, I can open a bug for it. But yes I think with gthree 0.9.0 we need master of hexgl as it is split into gtk3 https://github.com/alexlarsson/gnome-hexgl/commit/edf6a7592ce1cab6b936b0dfb8c780d3fe710d3e but this hasn't made into a release.

Reply:
if the ghtree update cause this then it should be held back.

Reply:
It looks unrelated to gthree, it can't find the headers `./src/hexgl.c:2:10: fatal error: gtk/gtk.h: No such file or directory`, but these are present which is weird.

Reply:
Ok I opened an issue about the build failure and specified a commit which works fine.

Reply:
so hexgl 0.2.0 build with gthree 0.2.0.

As I said, sometime you should just hold back upgrading things.

Reply:
See #3.

Reply:
Yea, I realized that later when I opened the issue for HexGL. The error was throwing me off.